### ISSUE TITLE
System.ServiceModel.Duplex and System.ServiceModel.Security shipped reference assembly revisions over what was in desktop

### ISSUE THREAD

**Issue by ericstj** [dotnet/wcf#1839]
These two packages contain reference assemblies with revisions over what was first shipped in desktop.

The result is that you compile a netstandard1.1 library then try to use that in a net45 project and the versions don't match.

Roslyn appears to be tolerant of the mismatch, and using dotnet CLI I don't see a warning or error from RAR, but I do see the following in the log:
```
                     Dependency "System.ServiceModel.Security, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". (TaskId:52)
                         Could not resolve this reference. Could not locate the assembly "System.ServiceModel.Security, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. (TaskId:52)
                             For SearchPath "F:\wcfTest\lib\bin\Debug\netstandard1.1". (TaskId:52)
                             Considered "F:\wcfTest\lib\bin\Debug\netstandard1.1\System.ServiceModel.Security.winmd", but it didn't exist. (TaskId:52)
                             Considered "F:\wcfTest\lib\bin\Debug\netstandard1.1\System.ServiceModel.Security.dll", but it didn't exist. (TaskId:52)
                             Considered "F:\wcfTest\lib\bin\Debug\netstandard1.1\System.ServiceModel.Security.exe", but it didn't exist. (TaskId:52)
                             For SearchPath "{TargetFrameworkDirectory}". (TaskId:52)
                             Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\System.ServiceModel.Security.winmd", but it didn't exist. (TaskId:52)
                             Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\System.ServiceModel.Security.dll", but it didn't exist. (TaskId:52)
                             Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\System.ServiceModel.Security.exe", but it didn't exist. (TaskId:52)
                             Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\Facades\System.ServiceModel.Security.winmd", but it didn't exist. (TaskId:52)
                             Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\Facades\System.ServiceModel.Security.dll",
                                                              but its name "System.ServiceModel.Security, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"
                                                              didn't match the expected name "System.ServiceModel.Security, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a". (TaskId:52)
                             Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\Facades\System.ServiceModel.Security.exe", but it didn't exist. (TaskId:52)
                         Required by "F:\wcfTest\lib\bin\Debug\netstandard1.1\lib.dll". (TaskId:52)
```

We went out of our way to fix this for corefx packages
https://github.com/dotnet/corefx/commit/6ad2e34491d3cd3a078c225f98b2c49268db81e1

We did this because of the issues I outlined here:
https://github.com/dotnet/corefx/issues/5842

Now some of these appear to be fixed in the CLI but you may want to test this scenario in VS 2017 and potentially downlevel.

I uncovered this problem when I made some changes to the static validation of packages and started catching this.  I've had to add a suppression to permit your packages to build with this issue.  See https://github.com/dotnet/buildtools/commit/c0dc59cd69abc219afe1e3be741c239234dad1f0.

/cc @MattGal

**Comment by zhenlan** [comment]
@StephenBonikowsky can you please work with @ericstj and @MattGal to understand what needs to be done here?

**Comment by StephenBonikowsky** [comment]
Looking into it.

**Comment by zhenlan** [comment]
Fixed by #1894
